src/main/java/by/itacademy/javaenterprise/knyazev/services/IService.java

```diff
@@ -0,0 +1,8 @@
+package by.itacademy.javaenterprise.knyazev.services;
+
+import java.util.List;
+
+public interface IService<T> {

```

Comment:
```suggestion
public interface Service<T> {

```
в java по code-convention `I` не пишут в интерфейсах